app/models/planet.py

```diff
@@ -0,0 +1,7 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(50), nullable=False)
```

Comment:
Setting a hard limit for Planet names? What if there's a Planet with a really really long name?!